### DISCUSSION TITLE
v7.1.0

### DISCUSSION THREAD

**Discussion by twelch** [Turfjs/turf#2699] [Announcements]
## What's Changed
* `@turf/distance` Simplify unnecessary union type in turfDistance by @isti115 in https://github.com/Turfjs/turf/pull/2618
* Upgrade to Lerna 8 by @mfedderly in https://github.com/Turfjs/turf/pull/2612
* Deprecate CHANGELOG, improve release action, PR template, Contributing doc by @twelch in https://github.com/Turfjs/turf/pull/2621
* `@turf/helpers` Reduce bundle size of apps that import @turf/helpers by @smallsaucepan in https://github.com/Turfjs/turf/pull/2623
* `@turf/line-overlap` change deep-equals dependency for smaller bundle size by @mfedderly in https://github.com/Turfjs/turf/pull/2631
* `@turf/mask` Converted turf-mask to Typescript and added support for geometry parameters by @smallsaucepan in https://github.com/Turfjs/turf/pull/2644
* `@turf/midpoint` Converted turf-midpoint to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2645
* `@turf/planepoint` Converted turf-planepoint to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2646
* `@turf/square` Converted turf-square to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2648
* `@turf/standard-deviational-ellipse` Converted turf-standard-deviational-ellipse to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2649
* Pin pnpm version using corepack by @RobinVdBroeck in https://github.com/Turfjs/turf/pull/2602
* `@turf/tesselate` Converted turf-tesselate to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2650
* `@turf/transform-rotate` Converted turf-transform-rotate to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2651
* `@turf/transform-scale` Converted turf-transform-scale to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2652
* `@turf/transform-translate` Converted turf-transform-translate to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2653
* `@turf/unkink-polygon` Converted turf-unkink-polygon to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2654
* `@turf/voronoi` Converted turf-voronoi to Typescript by @smallsaucepan in https://github.com/Turfjs/turf/pull/2655
* `@turf/boolean-intersects` Fix boolean-intersects docs by @izzybps in https://github.com/Turfjs/turf/pull/2593
* Updated documentation.js to resolve critical parse-url vulnerability by @smallsaucepan in https://github.com/Turfjs/turf/pull/2664
* `@turf/area`: bug fix: off-by-one bug in area calculation by @Abdumbo99 in https://github.com/Turfjs/turf/pull/2665
* Upgrade dependencies by @mfedderly in https://github.com/Turfjs/turf/pull/2668
* `@turf/isobands` `@turf/isolines` Remove unused matrix-to-grid dependency by @mfedderly in https://github.com/Turfjs/turf/pull/2669
* `@turf/random` fix randomPolygon generating polygons outside of the given `bbox` by @nnmrts in https://github.com/Turfjs/turf/pull/2659
* `@turf/boolean-intersects` `@turf/booean-disjoint` Expose options parameter to ignore self intersections. by @01100100 in https://github.com/Turfjs/turf/pull/2636
* `@turf/boolean-valid` fix checkClosingRing to check for polygon being closed by @superDoss in https://github.com/Turfjs/turf/pull/2563
* `@turf/helpers` Implemented azimuthToBearing by @basvdijk in https://github.com/Turfjs/turf/pull/2674
* Added support/6.x branch to CI config by @smallsaucepan in https://github.com/Turfjs/turf/pull/2673
* `@turf/nearest-point-on-line` include index of MultiLineString geometries in nearest-point-on-line by @andrewharvey in https://github.com/Turfjs/turf/pull/2574
* `@turf/point-to-line-distance` Fix two small typos by @mfedderly in https://github.com/Turfjs/turf/pull/2675
* Update PR/develop action to cache pnpm files by @mfedderly in https://github.com/Turfjs/turf/pull/2671
* Add explicit geojson types dependency by @mfedderly in https://github.com/Turfjs/turf/pull/2676
* Rework prettier setup by @mfedderly in https://github.com/Turfjs/turf/pull/2677
* Update @types/geojson to 7946.0.10 minimum by @mfedderly in https://github.com/Turfjs/turf/pull/2688
* `@turf/mask` Stop turf-mask mutating by default, make it an option by @farkmarnum in https://github.com/Turfjs/turf/pull/2635
* Add test.example.js to .prettierignore by @mfedderly in https://github.com/Turfjs/turf/pull/2689
* `@turf/mask` Fix @turf/mask benchmarks to exclude test fixtures that are not usable by @mfedderly in https://github.com/Turfjs/turf/pull/2692
* `@turf/kinks` Revert @turf/kinks to 6.5.0 version by @mfedderly in https://github.com/Turfjs/turf/pull/2693
* `@turf/cluster-dbscan`Update cluster-dbscan docs by @mwenko in https://github.com/Turfjs/turf/pull/2624
* `@turf/*-grid` `@turf/area` `@turf/helpers` Clarify behavior of some existing turf modules by @twelch in https://github.com/Turfjs/turf/pull/2683
* Upgrade pnpm/action-setup in github actions by @mfedderly in https://github.com/Turfjs/turf/pull/2696

## New Contributors
* @isti115 made their first contribution in https://github.com/Turfjs/turf/pull/2618
* @izzybps made their first contribution in https://github.com/Turfjs/turf/pull/2593
* @Abdumbo99 made their first contribution in https://github.com/Turfjs/turf/pull/2665
* @nnmrts made their first contribution in https://github.com/Turfjs/turf/pull/2659
* @01100100 made their first contribution in https://github.com/Turfjs/turf/pull/2636
* @superDoss made their first contribution in https://github.com/Turfjs/turf/pull/2563
* @basvdijk made their first contribution in https://github.com/Turfjs/turf/pull/2674
* @andrewharvey made their first contribution in https://github.com/Turfjs/turf/pull/2574
* @farkmarnum made their first contribution in https://github.com/Turfjs/turf/pull/2635

**Full Changelog**: https://github.com/Turfjs/turf/compare/v7.0.0...v7.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Turfjs/turf/releases/tag/v7.1.0'>v7.1.0</a>.</em>

**Comment by twelch** [comment]
A new snapshot+update of the docs at turfjs.org will be forthcoming.  Please be patient.  We're still dialing that in.

**Reply by adamchainz** [reply]
Thanks. Are you aware the docs site has been down since around mid October? https://web.archive.org/web/*/https://www.turfjs.org/

**Reply by basvdijk** [reply]
@adamchainz It only seems the `www` redirect is not working. https://turfjs.org/ is working fine

**Reply by smallsaucepan** [reply]
Thanks @adamchainz  and @basvdijk. This is a known issue with the website setup from way back: https://github.com/Turfjs/turf-www/issues/139

Perhaps ask on that issue if adding the www domain might be possible.
